includes/helpers/class-listing-display-helper.php

```diff
@@ -96,7 +96,7 @@ public static function single_listing_vars( $include = array() ) {
 			}
 			$vars['listing_css_class'] .= ' with-image';
 		}
-
+		$vars['listing_has_excerpt'] = $vars['fields']->has_excerpt();
```

Comment:
Can we handle this in the WPBDP_Field_Display_List class too? Everything else is returned conditionally, so it looks like this may result in an unexpected response. Would $vars['fields'] always be set before this point?

Reply:
Looks great!